web/resources/user/views/UserProfile.js.jsx

```diff
@@ -89,9 +75,10 @@ class UserProfile extends Binder {
   render() {
     const { user } = this.props;
 
-    let pictureUrl = user.profilePicUrl || '/img/defaults/profile.png';
+    const pictureUrl = user.profilePicUrl || '/img/defaults/profile.png';
```

Comment:
I'd like to drop the default profile image and use the user's first/last initials like we do in ImagineTime

Reply:
Probably in a separate pull request though